### PR TITLE
Fix camel case module

### LINKED ISSUE #703: Consistent module naming
I don't want to be the bad guy 😄 , but there is a module file named with camelCase. Disregard me if this is not a problem.

### PR DESCRIPTION
**Pull request by seemethere**
Closes #703

### CODE REVIEW

**Comment by r0fls** [comment]
i personally don't see any benefit to keeping this outside of sanic/log.py

File: docs/sanic/logging.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ By default, log_config parameter is set to use sanic.config.LOGGING dictionary f
 
 And `filters`:
 
-- accessFilter (using sanic.default_filter.DefaultFilter)<br>
+- accessFilter (using sanic.logging.DefaultFilter)<br>
```
Comment (r0fls) [inline docs/sanic/logging.md]: I think this needs to be `using sanic.log.DefaultFilter`

File: sanic/config.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
-from sanic.defaultFilter import DefaultFilter
 import os
 import sys
 import syslog
 import platform
 import types
 
+from sanic.logging import DefaultFilter
```
Comment (r0fls) [inline sanic/config.py]: whoops, here too :)

Comment (seemethere) [inline sanic/config.py]: I feel like we should have a test for this somewhere...

**Pull request merged**